src/runtime/server/plugin.ts

```diff
@@ -7,7 +7,7 @@ import { loadVueI18nOptions, loadInitialMessages, makeFallbackLocaleCodes, loadL
 
 import type { NitroAppPlugin } from 'nitropack'
 import type { H3Event } from 'h3'
-import type { NuxtApp } from 'nuxt/dist/app'
+import type { NuxtApp } from 'nuxt/app'
```

Comment:
In this case, `#app` is only meant to be imported in the Vue part of the app, not the Nitro part, so `#app` doesn't have any meaning there. But, since it's just a type import, it's okay to keep - but in that case we can use the subpath `nuxt/app`.